packages/amplify-util-uibuilder/src/commands/utils/storageManager.ts

```diff
@@ -0,0 +1,18 @@
+import { StudioForm, StudioSchema } from '@aws-amplify/codegen-ui';
+
+export const hasStorageField = (form: StudioForm): boolean => {
+  const result = Object.values(form.fields).some((field) => {
+    return 'inputType' in field && field.inputType?.type === 'StorageField';
+  });
+
+  return result;
+};
+
+export function isFormSchema(schema: StudioSchema): schema is StudioForm {
+  return (
+    schema &&
+    Object.keys(schema).find(function (key) {
+      return key === 'fields';
+    }) !== undefined
+  );
```

Comment:
nit: I prefer a more succinct style with arrow functions

```suggestion
  return schema && Object.keys(schema).find((key) => key === 'fields') !== undefined;
```

Reply:
cleaned up the helper functions utilizing guard functions and `includes`